install_config/install/advanced_install.adoc

```diff
@@ -1601,7 +1614,7 @@ Infrastructure].
 ====
 
 Note the following when using the `native` HA method:
-
+Note
```

Comment:
Was this purposeful? It looks like it might be an accidental add

Reply:
Removed it. Thanks.